solidity/contracts/PhasedEscrow.sol

```diff
@@ -0,0 +1,73 @@
+pragma solidity 0.5.17;
+
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+import "openzeppelin-solidity/contracts/token/ERC20/IERC20.sol";
+import "openzeppelin-solidity/contracts/token/ERC20/SafeERC20.sol";
+
+interface IBeneficiaryContract {
+    function __escrowSentTokens(uint256 amount) external;
+}
+
+// @title PhasedEscrow
```

Comment:
We should have three backslashes `///`.

Reply:
Done in https://github.com/keep-network/keep-core/pull/2124/commits/ad07a24984b4c4e53b353d2ab816333ea2de0d07